### PR TITLE
 updated readme

### PR DESCRIPTION
**Pull request by tahminaringer**

### CODE REVIEW

**Comment by tahminaringer** [comment]
- [ x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ x] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [x ] Picture of whiteboard

**Comment by tahminaringer** [comment]
# Singly Linked List (labs 5 and 6)

- By Tahmina Ringer, Jeremy Penning, Ashley Moore
- Created a linked list and tests

## Challenge

### Create a Node class-

1. Method of insert adds a new value to the head of the list
2. Method of Includes takes in a value and returns a boolean it exist or not
3. Method toString returns a string of all the values in the linked list
4. Method of append that adds a new node with a value to the end of the node list
5. Method of insertBefore finds a value in the LL and adds a new value before the found value
6. Method of insertAfter finds a value in the LL and adds a new value after the found value

Input: value, no argument
Output: Boolean, string of values, new value appended to the end of the list, new value inserted into the LL before or after the found existing value

## Aproach & Efficiency

My approach was to start with the code and then write the tests.

Big 0 = O(1), O(n)

## Resources

- I utilized my classmates for this
- [geeksforgeeks](https://www.geeksforgeeks.org/implementation-linkedlist-javascript/);

[URL code challenge 05](linked-list.png)
[URL code challenge 06](URL2.png)

## API

- insert- to add a new node
- includes- to search for existing node
- toString- to return a string of all the nodes
- append - add new node to LL tail
- insertBefore- find existing value and insert node before existing value
- insertAfter- find existing value and insert node after existing value

File: javascript/code-challenges/linkedList/README.md [hunk 1]
```diff
@@ -36,3 +36,6 @@ Big 0 = O(1), O(n)
 - insert- to add a new node
 - includes- to search for existing node
 - toString- to return a string of all the nodes
+- append - add new node to LL tail
+- insertBefore- find existing value and insert node before existing value
+- insertAfter- find existing value and insert node after existing value
```
Comment (adrianhuebner) [inline javascript/code-challenges/linkedList/README.md:41]: I would love to see some code and tests written out for this!
